### ISSUE TITLE
[pt-br] Sync task-tutorial-prereqs.md in Brazilian Portuguese with the English version

### ISSUE THREAD

**Issue by stormqueen1990** [kubernetes/website#39037]
### What would you like to be added

Update the translation of the `task-tutorial-prereqs.md` partial to match the latest English version.

Atualizar a tradução do parcial `task-tutorial-prereqs.md` para refletir a versão atual da página em inglês.

File path in repository/caminho do arquivo no repositório: `content/pt-br/includes/task-tutorial-prereqs.md`

### Why is this needed

To synchronize the translated partial with the current English version.

Para sincronizar o parcial traduzido com a última versão em inglês.

#### How to test

This is a partial included as a prerequisite section on top of the task pages. To verify this change, re-build the website (instructions [here](https://github.com/kubernetes/website#prerequisites)) and navigate to a task page in Brazilian Portuguese (for example, https://kubernetes.io/pt-br/docs/tasks/configmap-secret/managing-secret-using-kubectl/). The text at the top in the section "Antes de você começar" should contain the same instructions as the section in English.

#### Como testar

Este é um parcial incluído na forma de seção de pré-requisitos no topo das páginas de tarefas. Para verificar esta alteração, reconstrua o website (conforme instruções [aqui](https://github.com/kubernetes/website/blob/main/README-pt.md#pr%C3%A9-requisitos)) e navegue para uma página de tarefa em português brasileiro (por exemplo, https://kubernetes.io/pt-br/docs/tasks/configmap-secret/managing-secret-using-kubectl/). O texto no topo da página, na seção "Antes de você começar" deve ser equivalente em instruções à sua versão em inglês.

#### Creating a pull request

When creating a pull request for this issue, please name it:
```
[pt-br] Update includes/task-tutorial-prereqs.md
```

In the comment section of the pull request, please add ["fixes" followed by the number of this issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

#### Criando um pull request

Ao criar um pull request para esta issue, por favor coloque como título:
```
[pt-br] Update includes/task-tutorial-prereqs.md
```

No comentário do seu pull request, por favor escreva ["fixes" seguido do número desta issue](https://docs.github.com/pt/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

/language pt
/good-first-issue
/help

**Comment by redone9211** [comment]
i would like to contribute to this issue, can you let me know how i can solve it.

**Comment by divaspathak** [comment]
I would like to contribute to this issue, can you please assign it to me?

**Comment by stormqueen1990** [comment]
> I would like to contribute to this issue, can you please assign it to me?

Hi there, @divaspathak! Since you have previously commented on this issue, [you can use the `/assign` command to assign it to yourself](https://prow.k8s.io/command-help?repo=kubernetes%2Fwebsite).

Please note that this issue requires translation of content from English to Brazilian Portuguese.

**Comment by Aakarshan-369** [comment]
Hi @stormqueen1990 I am having issues with the tests in PR #39062. I worked on this as a good first issue and would appreciate any feedback. Thanks! :)

**Comment by stormqueen1990** [comment]
Solved via #39062.
